README.md

```diff
@@ -0,0 +1,134 @@
+# Mattermost build
+
+Jenkins automation for building mattermost
+
+### Command to hook up new kuberntes cluster to docker repository
+
+```
+kubectl create secret docker-registry myregistrykey --docker-server=mmjenkins.azurecr.io --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=dev-ops@mattermost.com
+```
+
+# Deploying the build server locally
+
+## Prerequisites
+- virtualization platform
+- docker
+- install `minikube`, `kubectl`, `helm` by following instructions here:
+  - [minikube](https://github.com/kubernetes/minikube/releases)
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - [helm](https://docs.helm.sh/helm/#helm-install)
+
+## Overview
+
+- startup minikube cluster
+- deploy mattermost-build (Jenkins)
+- run builds
+- destroy minikube cluster
+
+## Setup
+
+```
+minikube start
+```
+This will start a `minikube` environment with a default of 2GB and 2 cpus. These values can be set like this:
+```
+minikube start --memory 8192 --cpus 4 --mount
+```
+Config is stored in `~/.minikube`
+
+enable ingress
+```
+minikube addons enable ingress
+```
+
+Add `helm` repos
+```
+
```

Comment:
this is intentionally left blank?

Reply:
Forgot about that. I couldn't remember if I needed to add any repos or if they were there by default.

Reply:
I followed the steps and did not add anything here

Reply:
I'll add a commit to remove this line.